### DISCUSSION TITLE
General Comments Thread

### DISCUSSION THREAD

**Discussion by pwdel** [openpredictionmarkets/socialpredict#149] [General]
This thread is for anyone on the entire web who may have a comment, question or idea.

**Comment by pesicsasa** [comment]
Hey, 
Is there a sample UI deployed to test this out or screenshots/videos?

**Reply by pwdel** [reply]
Greetings @pesicsasa yes, you can view staging here. I can set up a username and password for you if you would like to test it out.

https://brierfoxforecast.com/

**Reply by pwdel** [reply]
@pesicsasa If you would like, if Discord works for you, you could join our Discord here and I can get it to you by direct message (DM). https://discord.gg/vtGQBwwd

**Reply by raisch** [reply]
The Discord invite has expired. Can I get an invite? Thanks.

**Reply by pwdel** [reply]
https://discord.gg/BvjtfESjmJ ... this one should be permanent

**Comment by Pat222222** [comment]
Hi, kinda random but I am developing a discord bot Predictions Market but am lookimg to potentially implement one on a testnet on a Bitcoin Layer 2 Network. Is this feasible with your codebase?

**Reply by pwdel** [reply]
Greetings, hey thanks for your question, no problem at all for the randomness. I'm not familiar enough with Bitcoin Layer 2 Network or how to create a Discord Bot to say whether the necessary features would be on our roadmap.

What I can say is that our design philosophy includes the capability to run a real money system, e.g. that points would not be created or destroyed.

Are you looking to contribute and help build out something public or are you just looking for something that works right now, immediately?

Give us a star/follow, maybe if what we have doesn't work for you today, maybe it would in the future.

**Comment by Pat222222** [comment]
Hey, I have actually already created the discord PM bot! Im not much of a developer but wanted to play with potentially launching a PM on a Bitcoin Layer 2 for fun.

**Reply by pwdel** [reply]
That's cool!

**Comment by markokovac16** [comment]


**Reply by pwdel** [reply]
Done, sorry for the delay.
